.travis.yml

```diff
@@ -14,6 +14,7 @@ install:
 - ./install_ta-lib.sh
 - export LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH
 - pip install --upgrade flake8 coveralls pytest-random-order pytest-asyncio mypy
+- pip install future
```

Comment:
don't think this is needed with the latest `py_find_1st` version - which dropped this install dependency